src/web/core/src/main/java/org/geoserver/web/wicket/CRSPanel.java

```diff
@@ -267,6 +268,11 @@ String toSRS(CoordinateReferenceSystem crs) {
         try {
             if (crs != null) {
                 Integer epsgCode = CRS.lookupEpsgCode(crs, false);
+                String srs = srsTextField.getModelObject();
```

Comment:
This looks messy, the text field now returns ids prefixed in some cases, but not in others? Can always you return the srs  with a prefix instead of duplicating prefixing logic around?

Reply:
If I understood correctly, I should always return "EPSG:XXXX" even if it was initially an URN OGC?

Reply:
No, I just mean it would seem better if the text field either contained "EPSG:XYZW" or "urn.....::XYZW" but not just "XYZW", the prefixing logic I'm seeing seems to be caring for the last case.

Reply:
ok now I understood, so if its not a URN , we simply return the srs string if it is already prefixed with EPSG:

Reply:
so

`srs != null && srs.startsWith(BasicResourceConfig.URN_OGC_PREFIX)`

will become 

`srs != null && (srs.startsWith(BasicResourceConfig.URN_OGC_PREFIX) || srs.startsWith("EPSG:"))`

Reply:
Yep. Uniform handling for the two cases (including a dedicated constant for the "EPSG" prefix):